R/common.R

```diff
@@ -122,6 +123,28 @@ runJaspResults <- function(name, title, dataKey, options, stateKey, functionCall
   }
 }
 
+registerFonts <- function() {
+
+  if (requireNamespace("ragg") && requireNamespace("systemfonts")) {
+
+    # To register custom font files shipped with JASP we need the path to the font file.
+    # Next the font could be loaded like this:
+    #
+    # fontName <- "FreeSansJASP"
+    # fontFile <- "~/github/jasp-desktop/Desktop/resources/fonts/FreeSans.ttf"
+    # systemfonts::register_font(fontName, normalizePath(fontFile))
+    # jaspGraphs::setGraphOption("family", fontName)
+
+    if (exists(".resultsFont"))
+      jaspGraphs::setGraphOption("family", .resultsFont)
```

Comment:
And in fact `jaspGraphs`? 

For renv I have removed the default installation of jaspGraphs in the delivered R-library. Because renv  will install the particular version needed by each pkg. Would that break this code?

Reply:
I guess it would actually make more sense to store the desired font in `jaspBase` and then have `jaspGraphs` query that. This way we make the dependency-graph a bit more linear.